src/css-token-stream-value.js

```diff
@@ -0,0 +1,69 @@
+(function(internal, scope) {
+
+    function CSSTokenStreamValue(values) {
+        if (values == undefined) {
+            values = [];
+        }
+        if (!Array.isArray(values)) {
+            throw new TypeError('CSSTokenStreamValue should be an array of string or CSSVariableReferenceValue');
+        }
+        for (var i = 0; i < values.length; i++) {
+            if (typeof values[i] != 'string' && !(values[i] instanceof CSSVariableReferenceValue)) {
+                throw new TypeError("CSSTokenStreamValue's elements should be string or CSSVariableReferenceValue");
+            }
+        }
+        this.listOfReferences = values;
```

Comment:
This should be a "private" variable. In javascript, since you can't really have real private variables, we mark things as "private" by using an underscore at the start of the name:

```javascript
this._listOfReferences = values;
```